pkg/apb/rhcc_registry.go

```diff
@@ -127,7 +126,17 @@ func (r RHCCRegistry) imageToSpec(image *Image) (*Spec, error) {
 		return nil, err
 	}
 
+	if conf.Config == nil {
+		r.log.Error("Did not find v1 Manifest in image history. Skipping.")
+		return nil, errors.New("Error: v1 Manifest does not exist for this image")
```

Comment:
It doesn't look like there are any fatal errors in this function. I think it's a legitimate assumption that if there is an invalid apb  we 1) alert the user of the problem 2) skip loading that apb. With that assumption, we should only have to log errors, but not return any. So you can rid of error as a return. 

Let me know if folks agree with that assumption.